### DISCUSSION TITLE
Clear deprecation warnings console command

### DISCUSSION THREAD

**Discussion by mattsbanner** [craftcms/cms#10430] [Ideas] [answered]
It would be great if we could clear all deprecation warnings from a console command, if this was run on each deploy - it would keep the list relevant to the current codebase.

**Comment by brandonkelly** [comment, marked as answer]
Added for the next release! (adfb879c1ab8b0da105742884cdaef2a3688edc6)

**Reply by brandonkelly** [reply]
Craft 3.7.31 is out now with the new command:

```sh
php craft clear-deprecations
```
